src/promptflow-evals/promptflow/evals/evaluators/_content_safety/_hate_unfairness.py

```diff
@@ -2,13 +2,12 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 from promptflow._utils.async_utils import async_run_allowing_running_loop
+from promptflow.evals.common.constants import EvaluationMetrics
 
 try:
-    from .common import ContentSafetyEvaluatorBase
-    from .common.constants import EvaluationMetrics
+    from ._content_safety_base import ContentSafetyEvaluatorBase
```

Comment:
Any downside if we also move content safety base to common?

Reply:
I considered that - my main reason against was that unlike the other files, which can all be extended to be used outside of a content safety content, the Content Safety Base is explicitly tied to the content safety evals, so having it remain among them seemed logically sound.